addon/adapters/osf-adapter.js

```diff
@@ -21,12 +21,17 @@ import {
  * @uses GenericDataAdapterMixin
  */
 export default DS.JSONAPIAdapter.extend(GenericDataAdapterMixin, {
-    headers: {
-        ACCEPT: 'application/vnd.api+json; version=2.6'
-    },
+    currentUser: Ember.inject.service('current-user'),
```

Comment:
Do we have to do `Ember.inject.service('current-user')` or is `Ember.inject.service()` sufficient? I think if the `Ember.inject.service()` is called with no arguments, it would automatically use the dasherized property name to look up the service.